Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "typst"
 version = "0.0.0"
 authors = ["The Typst Project Developers"]
 edition = "2021"
+resolver = "2"
```

Comment:
This is not default for workspaces! It is however default for new rust projects.